upcloud/resource_upcloud_objectstorage_test.go

```diff
@@ -15,27 +15,24 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 	"github.com/minio/minio-go/v7"
+	"github.com/rs/xid"
 )
 
 const expectedDescription = "My object storage"
 const expectedZone = "pl-waw1"
 const expectedKey = "an access key"
 const expectedSecret = "a secret key"
+const runPrefix = "testacc-"
 
-const expectedName1 = "test-name1"
-const expectedName2 = "test-name2"
-const expectedName3 = "test-name3"
+var runID = xid.New().String()
+var expectedName1 = fmt.Sprintf("%s%s-1", runPrefix, runID)
```

Comment:
Should we prefix these `upcloud` package level variables with eg `objectStorageTest..` ? Not a big deal but might be confusing when editor suggests `expectedName1` when working on eg server resource :)

Reply:
Yes 👍